core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -99,23 +98,26 @@ public void createNamespace(String namespace, Map<String, String> options)
   public void createTable(
       String namespace, String table, TableMetadata metadata, Map<String, String> options)
       throws ExecutionException {
-    if (!rdbEngine.isValidNamespaceOrTableName(table)) {
-      throw new ExecutionException("The table name is not acceptable: " + table);
-    }
-
     try (Connection connection = dataSource.getConnection()) {
-      createTableInternal(connection, namespace, table, metadata);
-      createIndex(connection, namespace, table, metadata);
-      addTableMetadata(connection, namespace, table, metadata, true);
+      createTableInternal(connection, namespace, table, metadata, false);
+      addTableMetadata(connection, namespace, table, metadata, true, false);
     } catch (SQLException e) {
       throw new ExecutionException(
           "Creating the " + getFullTableName(namespace, table) + " table failed ", e);
     }
   }
 
-  private void createTableInternal(
-      Connection connection, String schema, String table, TableMetadata metadata)
-      throws SQLException {
+  @VisibleForTesting
+  void createTableInternal(
```

Comment:
I made this method package private to avoid replicating unit tests since several public methods use this method.